app/src/main/java/de/westnordost/streetcomplete/data/meta/CountryInfo.kt

```diff
@@ -0,0 +1,116 @@
+package de.westnordost.streetcomplete.data.meta
+
+import de.westnordost.streetcomplete.osm.LengthUnit
+import de.westnordost.streetcomplete.quests.max_speed.SpeedMeasurementUnit
+import de.westnordost.streetcomplete.quests.max_weight.WeightMeasurementUnit
+import kotlinx.serialization.Serializable
+import java.util.Locale
+
+@Serializable
+data class IncompleteCountryInfo(
+    // this value is not defined in the yaml file but it is the ISO language code part of the file name!
+    // i.e. US for US-TX.yml
+    val countryCode: String,
+
+    // sorted alphabetically for better overview
+    val additionalStreetsignLanguages: List<String> = emptyList(),
```

Comment:
Wouldn't it be easier to make all fields in `IncompleteCountryInfo` nullable instead of assigning some an empty list? Then, in `CountryInfo` below, you can just use 

```kotlin
additionalStreetsignLanguages: List<String> get() = 
    infos.firstNotNullOf { it.additionalStreetsignLanguages } ?: emptyList()
```

etc.